### PR TITLE
Add more node types as destinations to dashboards_flow_paths mv; Fix …

### PR DESCRIPTION
**Pull request by tiagojsag**
Add more node types as destinations to dashboards_flow_paths mv; Fix rubocop execution through git hooks

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -1119,6 +1126,202 @@ CREATE SEQUENCE public.country_properties_id_seq
 ALTER SEQUENCE public.country_properties_id_seq OWNED BY public.country_properties.id;
 
 
+--
+-- Name: dashboard_template_commodities; Type: TABLE; Schema: public; Owner: -
```
Comment (agnessa) [inline db/structure.sql]: I think we have stuff that doesn't belong here? One way to keep this file limited only to changes introduced in current branch is to 1. reset it to be a copy of base branch 2. `RAILS_ENV=test rake db:drop db:create db:structure:load` 3. `RAILS_ENV=test rake db:migrate`

Comment (tiagojsag) [reply db/structure.sql]: Yeah, I went down the lazy path, as always :P will do that and push my changes

**Comment by tiagojsag** [comment]
There, all laziness removed :muscle:

File: db/views/dashboards_flow_paths_mv_v02.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    a.node_id,
+    a."position"
+  FROM flows, LATERAL unnest(flows.path) WITH ORDINALITY a(node_id, "position")
+) flow_paths
+JOIN contexts ON flow_paths.context_id = contexts.id
+JOIN nodes ON flow_paths.node_id = nodes.id
+JOIN node_types ON nodes.node_type_id = node_types.id
+JOIN context_node_types cnt ON node_types.id = cnt.node_type_id AND flow_paths.context_id = cnt.context_id
+JOIN context_node_type_properties cnt_props ON cnt.id = cnt_props.context_node_type_id
+WHERE cnt_props.column_group = 0 OR node_types.name IN ('COUNTRY', 'IMPORTER', 'EXPORTER', 'TRADER');
```
Comment (agnessa) [inline db/views/dashboards_flow_paths_mv_v02.sql]: sorry I missed this - this condition needs to be changed as well to retrieve correct row: `WHERE cnt_props.column_group IN (0,3) OR node_types.name IN ('IMPORTER', 'EXPORTER', 'TRADER');`

**Comment by tiagojsag** [comment]
@agnessa ready for re-re-review

**Pull request merged**